x-pack/platform/packages/shared/kbn-slo-schema/src/schema/indicators.ts

```diff
@@ -28,7 +28,7 @@ const filtersSchema = t.array(
         key: t.string,
         field: t.string,
         params: t.any,
-        value: t.string,
+        value: t.union([t.string, t.record(t.string, t.string)]),
```

Comment:
🐞 When using filters, the value can be a string like "500" or an object like `{ "lt": "500" }` when using less than operator.